sdk/cognitiveservices/cognitiveservices-inkrecognizer/package.json

```diff
@@ -60,35 +60,35 @@
   "private": true,
   "dependencies": {
     "@azure/abort-controller": "1.0.0-preview.2",
-    "@azure/core-arm": "1.0.0-preview.3",
+    "@azure/core-arm": "1.0.0-preview.4",
```

Comment:
We are planning to release the next version for core-arm and core-http soon.
The changes are in master branch.
Please update the dependencies here appropriately

Reply:
done